acquire/tools/decrypter.py

```diff
@@ -343,19 +343,16 @@ def main():
     if not progress:
         log.info("`rich` is not installed, progress will not be shown")
 
-    if args.output and len(args.files) > 1:
+    if args.output and len(args.files) > 1 and not args.files[0].is_file():
         parser.exit("--output is only allowed when decrypting a single file")
```

Comment:
In the new situation, this is no longer correct right? I think the test should be that `args.output` is a directory if `len(args.files) > 1` or if `len(args.files) == 1 and args.files[0].is_dir()`